packages/stargate/src/queries/utils.ts

```diff
@@ -9,11 +12,31 @@ export function toAccAddress(address: string): Uint8Array {
   return Bech32.decode(address).data;
 }
 
-/**
- * Use this to convert a protobuf.js class to the interface (e.g. Coin to ICoin)
- * in a ways that makes Jasmine's toEqual happy.
- */
-// eslint-disable-next-line @typescript-eslint/ban-types
-export function toObject<I extends object>(thing: I): Omit<I, never> {
-  return { ...thing };
+export function createPagination(
+  paginationKey?: Uint8Array,
+): {
+  readonly key: Uint8Array;
+  readonly offset: Long;
+  readonly limit: Long;
+  readonly countTotal: boolean;
+} {
+  return {
+    key: paginationKey ?? new Uint8Array(),
+    offset: Long.fromNumber(0),
+    limit: Long.fromNumber(0),
+    countTotal: false,
+  };
+}
+
+interface Rpc {
```

Comment:
Can we make this type public, since it is part of the public `createRpc` signature? Any maybe a little more verbose.

Reply:
Probably! I didn't because of this comment that I don't fully understand: https://github.com/stephenh/ts-proto/blob/v1.53.0/src/generate-services.ts#L262-L270

Reply:
I wonder what `index.ts`s he's referring to. Maybe there are (optional) index files?

The interface is defined multiple times indeed, but it is always the same one.

Reply:
> maybe a little more verbose

Any suggestions?

Reply:
Unfortunately I hardly understand what this is and how it fits in the stack

Reply:
I think this is the kind of `index.ts` which is meant: https://github.com/stephenh/ts-proto/blob/v1.53.0/integration/barrel-imports/index.ts
This one doesn't involve the generated `Rpc` types specifically, but shows what would go wrong if they were exported from both `foo.ts` and `bar.ts`. I don't believe this index file has been generated.